src/components/proposal/ProposalTableOfficer.tsx

```diff
@@ -179,7 +176,7 @@ const ProposalTableOfficer: React.FC<ProposalTableOfficerProps> = ({
             <Visibility />
           </IconButton>
         </Tooltip>
-        <Tooltip title="Clone proposal">
+        {/* <Tooltip title="Clone proposal">
```

Comment:
I think we should not leave commented code. All the changes are stored in  SVC.

Reply:
You are right I forgot this one 😅